aimmo-game/tests/test_simulation/dummy_avatar.py

```diff
@@ -39,17 +39,17 @@ def damage(self, amount):
 
 
```

Comment:
I found weird stuff in dummy_avatar, I'll open issues for them as it's not fully related to that PR

Reply:
see #353

Reply:
:+1: